Firestore/core/src/remote/remote_event.h

```diff
@@ -41,6 +42,8 @@ class TargetData;
 
 namespace remote {
 
+enum class BloomFilterApplicationStatus { Success, Skipped, FalsePositive };
```

Comment:
Please check with the rest of the code, but I think the naming convention for enum elements begins with lowercase "k", such as `kSuccess`, `kSkipped`, and `kFalsePositive`.